inc/taskjobview.class.php

```diff
@@ -478,7 +478,10 @@ public function ajaxModuleItemsDropdown($options)
                 $condition = ['id' => $filter_id];
             }
         }
-        $condition['is_active'] = 1;
+        $itemtype_class = new $itemtype();
+        if ($DB->fieldExists($itemtype_class->getTable(), 'is_active')) {
```

Comment:
You must use  `getItemForItemtype($itemtype)` instead of `new $itemtype()`. But instanciation here is probably not necessary since the `CommonDBTM::getTable()` is static.
```suggestion
        if ($DB->fieldExists($itemtype::getTable(), 'is_active')) {
```